tests/test_client.py

```diff
@@ -518,7 +524,7 @@ def test_extra_headers(
     request_mock: Mock,
     client: GGClient,
     session_headers: Any,
-    extra_headers: Optional[Dict[str, str]],
+    extra_headers: Dict[str, str] | None,
```

Comment:
We can't do this: we must support Python 3.7.

Besides, it's better to do these kind of changes in a separate PR, using a tool like pyupgrade to update all the code at once.